packages/core/src/tracing/index.js

```diff
@@ -197,6 +201,7 @@ exports.init = function init(_config, downstreamConnection, _processIdentityProv
         otelInstrumentations.init(config, cls);
       }
     }
+    iitmHook.init();
```

Comment:
questions:

1. Why do we init the iitm not in automaticTracingEnabled? 🤔 
2. Why do we call init for cjs apps? IMO we should not even require iitm when CJS app is used.

Reply:
updated the code